cargo-ledger/src/main.rs

```diff
@@ -202,9 +202,25 @@ fn build_app(
                     "--message-format=json-diagnostic-rendered-ansi",
                 ])
                 .args(&remaining_args)
-                .stdout(Stdio::piped())
-                .spawn()
-                .unwrap();
+                .stdout(Stdio::piped());
+
+            let c_sdk_path = match device {
+                Device::Nanos => std::env::var("NANOS_SDK"),
+                Device::Nanosplus => std::env::var("NANOSP_SDK"),
+                Device::Nanox => std::env::var("NANOX_SDK"),
+            };
+
+            match c_sdk_path {
+                Ok(path) => {
+                    cargo_cmd = cargo_cmd
+                        .arg(format!("--config env.LEDGER_C_SDK_PATH={}", path))
```

Comment:
Maybe check if `LEDGER_C_SDK_PATH` is not already set?

Reply:
Needed for deployement pipeline in fact, as `NANOS_SDK` and others will be set, but we don't want their values.

Reply:
ok done